internal/cli/main.go

```diff
@@ -190,8 +191,23 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 			log.fatalErr(migraterErr)
 		}
 
-		limit := -1
-		if flag.Arg(1) != "" {
+		limit := 0
+		if flag.Arg(1) == "" {
+			fmt.Println("Are you sure you want to apply all down migrations? [y/N]")
+
+			reader := bufio.NewReader(os.Stdin)
+			response, err := reader.ReadString('\n')
+			if err != nil {
+				log.fatal("error: not a valid response")
+			}
+			response = strings.ToLower(strings.TrimSpace(response))
+
+			if response == "y" {
```

Comment:
Since we're prompting the user, we should provide feedback on whether or not their response was excepted. e.g. we should print in both cases

Reply:
added a log in both cases